src/net.cpp

```diff
@@ -59,8 +64,9 @@ using std::set;
 
 const unsigned int 
     nStakeMaxAge = 90 * nSecondsPerDay,             //60 * 60 * 24 * 90; // 90 days as full weight
-    nOnedayOfAverageBlocks = (nSecondsPerDay / nStakeTargetSpacing)/10;  // the old 144
-    //nOnedayOfAverageBlocks = nSecondsPerDay / nStakeTargetSpacing;  // should be 144 if it's BTC!
+    //nOnedayOfAverageBlocks = (nSecondsPerDay / nStakeTargetSpacing)/10;  // the old 144
+    // <<<<<<<<<<<< test
+    nOnedayOfAverageBlocks = nSecondsPerDay / nStakeTargetSpacing;  // should be 144 if it's BTC!
```

Comment:
=> Good catch !!! It existed in the source code for a long time. Number of yacoin avarage blocks should be 1440, not 144.